grml.yaml

```diff
@@ -55,4 +55,58 @@ commands:
                 deps:
                     - human_pose_estimation_demo.build
                 exec: |
-                    build_napp human_pose_estimation_demo
\ No newline at end of file
+                    build_napp human_pose_estimation_demo
+
+    face_detection_mtcnn_demo:
```

Comment:
Deelte text & face detection parts. We are focusing solely on the pedestrain detection demo.